### DISCUSSION TITLE
Dataset feeds containing different links: identifying spatial extent

### DISCUSSION THREAD

**Discussion by bfrichet3** [INSPIRE-MIF/helpdesk#105] [Q&A] [answered]
Dear

I have a question about TG Recommandation 10 of I[NSPIRE TG regarding download services.](https://inspire.ec.europa.eu/documents/technical-guidance-implementation-inspire-download-services). As you may know, data providers are allowed to give access to subsets of one INSPIRE datasets instead of giving access to the all dataset, especially if it's a huge dataset (like cadastral parcels).

TG Recommandation 10 states that we can use bbox attribute to distinguish the spatial extent of files. At Belgian Cadastral Administration we have split the cadastral plans into 581 areas : one for each municipality.  Therefore it's quite difficult for me to establish bouding box of each municipality (we don't have such central repository) and it think it should be better to use the nationalID or the INSPIRE ID of the municipality (or to use any ID of any concerned administrative unit when they are used to split a dataset). 

How would you consider such a way of making the distinction between our links and how would you write it? I add a "sectionID" attribute in the link tag but I can use any other attribute. Moreover I added a "title" attribute with the official name of the municipality in the ATOM language.

Regards,

Benoît Fricheteau

Regards

**Comment by fabiovinci** [comment, marked as answer]
Dear @bfrichet3,

the use of the optional "bbox" attribute is suggested for the purposes of differentiating between files, where a dataset is provided in multiple physical files.
The criteria you used to split the datasets by the municipality is valid and the different files can be easily distinguished by "filename" and "title" since you used the official name of the municipality.

For this reason, I think you don't need other attributes.

**Reply by bfrichet3** [reply]
Dear @fabiovinci 

Thank you for your quick answer. In principle I agree with you but I don't like to use the municipality's name as an identifier in the Belgian context (as we have three official languages and as some muncipalities have three different official names).

Regards

**Reply by fabiovinci** [reply]
Ok, I mentioned the "official name of the municipality" since you wrote you used it for the title. If there are different names for the same municipality you could use the "Municipality codes" (NIS codes) that are unique and you can explain their use in the service metadata.
